<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="NavigationBar" name="NavBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 127);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HomePage" name="homePage" native="true"/>
   </item>
   <item>
    <widget class="TaskList" name="taskList" native="true"/>
   </item>
   <item>
    <widget class="ConfigPage" name="configPage" native="true"/>
   </item>
   <item>
    <widget class="ResultPage" name="resultPage" native="true"/>
   </item>
   <item>
    <widget class="SettingPage" name="settingPage" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationBar</class>
   <extends>QWidget</extends>
   <header>navigationbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaskList</class>
   <extends>QWidget</extends>
   <header>tasklist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultPage</class>
   <extends>QWidget</extends>
   <header>resultpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingPage</class>
   <extends>QWidget</extends>
   <header>settingpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigPage</class>
   <extends>QWidget</extends>
   <header>configpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>homepage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
